frameworks/keyed/imba/src/main.imba

```diff
@@ -0,0 +1,116 @@
+var A = ["pretty", "large", "big", "small", "tall", "short", "long", "handsome", "plain", "quaint", "clean", "elegant", "easy", "angry", "crazy", "helpful", "mushy", "odd", "unsightly", "adorable", "important", "inexpensive", "cheap", "expensive", "fancy"]
+var C = ["red", "yellow", "blue", "green", "pink", "brown", "purple", "brown", "white", "black", "orange"]
+var N = ["table", "chair", "house", "bbq", "desk", "car", "pony", "cookie", "sandwich", "burger", "pizza", "mouse",  "keyboard"]
+
+tag Row < tr
+	prop select
+	prop remove 
+	prop item
+	prop selected
+
+	def onSelect
+		@select(@item)
+
+	def onRemove 
+		@remove(@item)
+
+	def render
+		<self@{item:id} .selected=@selected>
+			<td.col-md-1> @item:id
+			<td.col-md-4><a :tap.onSelect> @item:label
+			<td.col-md-1><a :tap.onRemove> <span.glyphicon.glyphicon-remove :aria-hidden=true>
```

Comment:
When I checked in dev tools, `glyphicon` doesn't have `aria-hidden` attribute, so I guess it should be `<td.col-md-1><a :tap.onRemove> <span.glyphicon.glyphicon-remove aria-hidden=true>`, but it seems that Imba doesn't support aria attributes and I got an error `Uncaught TypeError: _1(...).flag(...).flag(...).flag(...).setAriaHidden is not a function`